lms/static/sass/shared/_course_object.scss

```diff
@@ -180,78 +191,116 @@
-        @include transition(all 0.15s linear 0s);
-        width: 100%;
-        overflow: hidden;
-
+      .course-image {
         .cover-image {
-          height: 200px;
+          height: 160px;
           overflow: hidden;
-          width: 100%;
+
+          &:before {
+            @include fade-transition();
+            content: '';
+            position: absolute;
+            top: 0;
+            left: 0;
+            background: black;
+            opacity: 0;
+            width: 100%;
+            height: 160px;
+            z-index: 10;
+          }
+
+          &:after {
+            @include fade-transition();
+            @include box-sizing(border-box);
+            @include calc( top, '80px - 25px');
+            @include calc( left, '50% - 100px');
+            content: 'LEARN MORE';
+            position: absolute;
+            width: 200px;
+            height: 50px;
+            z-index: 20;
+            border: 3px solid white;
+            border-radius: 3px;
+            font-family: $sans-serif;
+            font-weight: $font-semibold;
+            font-size: 1.1em;
+            line-height: 44px;
+            color: white;
+            padding: 0 20px;
+            text-align: center;
+            opacity: 0;
+          }
 
           img {
-            display: block;
-            min-height: 100%;
             width: 100%;
           }
         }
+      }
 
-        .desc {
-          @include box-sizing(border-box);
-          height: 120px;
-          overflow: hidden;
-          padding: 10px 10px 12px 10px;
-          position: relative;
-          width: 100%;
+      .course-info {
+        height: 200px;
 
-          section {
-            color: $base-font-color;
-            font: normal 1em/1.6em $serif;
-            margin: 0;
-            height: 100%;
-            overflow: hidden;
-            text-overflow: ellipsis;
-          }
+        .course-organization, .course-code, .course-date {
+          color: #3d3e3f;
+          font-size: 0.9em;
         }
 
-        .bottom {
-          @include box-sizing(border-box);
-          padding: 0px 10px 10px 10px;
-          width: 100%;
+        .course-organization {
+          padding: 11px 15px 2px;
+          line-height: 16px;
+        }
 
-          .university {
-            color: $lighter-base-font-color;
-            letter-spacing: 1px;
-          }
+        .course-code {
+          padding: 0 15px;
+          line-height: 24px;
+        }
 
-          .start-date  {
-            border-left: 1px solid $border-color-2;
-            margin-left: ($baseline/4);
-            padding-left: ($baseline/2);
-            color: $lighter-base-font-color;
-            letter-spacing: 1px;
-          }
+        .course-title {
+          height: 72px;
+          font-size: 1.1em;
+          color: #701f3f;
+          padding: 0 15px;
+          margin: 5px 0;
+          line-height: 24px;
+        }
+
+        .course-date {
+          margin-top: 30px;
```

Comment:
Please, when possible, calculate any spacing values (margin, padding, etc.) using the ``$baseline`` variable.